### ISSUE TITLE
tutorial on training?

### ISSUE THREAD

**Issue by hftsai** [lmb-freiburg/Unet-Segmentation#21]
Hi,
First, great work on the Nature methods paper.
I tried Unet on caffe when it first came out.

But I wonder how easy is it to prepare training data (with reference data) to further train Unet or finetune in the current imageJ?

Do you have a tutorial for this?

thank you

**Comment by ThorstenFalk** [comment]
From-scratch annotation is as easy as using the ROI tools in conjunction with the ROI manager in ImageJ. You can of course bootstrap the annotations with any other approach, e.g. the WEKA trainable segmentation and then only correct erroneous segmentations to obtain groundtruth data.

If you already have annotated data, you probably have mask annotations. You can also train from mask annotations, you just have to add them as overlay to the raw data (Image->Overlay->Add image (2D only), or Plugins->U-Net->Utilities->Embed Mask Annotations (2D and 3D)) . Probably labels must be incremented by 1 (Process->Math->Add... 1), because the U-Net segmentation plugin assumes 0 to be ignore and 1 to be background.

For further details check our annotation guide in the supplementary material of the NMeth paper.

**Comment by hftsai** [comment]
Hi Thorsten,
I missed the supplementary info. reading it now.
Thanks!
